doc/en/user/source/services/wms/global.rst

```diff
@@ -25,3 +25,13 @@ ENABLE_JSONP
 
 Can be ``true`` or ``false`` (defaults to: ``false``).
 When ``true`` the JSONP (text/javascript) output format is enabled.
+
+GETMAPKVPREQUESTREADER_TIMEOUT
```

Comment:
Same as bellow.

Reply:
Would it be worth to allow this configuration \ limits to be set on GeoServer UI? On the global configuration page or WMS service  since this looks specific?